### PR TITLE
Updated pt_BR strings for 4.11

### PR DESCRIPTION
**Pull request by Patola**

### CODE REVIEW

File: resources/i18n/pt_BR/fdmprinter.def.json.po [hunk 1]
```diff
@@ -1438,12 +1438,12 @@ msgstr "Conectar caminhos de contorno da base e topo quando estiverem próximos
...
 msgctxt "skin_monotonic label"
 msgid "Monotonic Top/Bottom Order"
-msgstr ""
+msgstr "Ordem Monotônica Superior/Inferior"
 
 #: fdmprinter.def.json
 msgctxt "skin_monotonic description"
 msgid "Print top/bottom lines in an ordering that causes them to always overlap with adjacent lines in a single direction. This takes slightly more time to print, but makes flat surfaces look more consistent."
-msgstr ""
+msgstr "Imprime filetes superiores e inferiores em uma ordem que os faz sempre se sobreporem a filetes adjacentes em uma única direção. Faz levar um pouco mais de tempo na impressão, mas torna superfícies planas mais consistentes."
```
[the remark this answers is not in the document]
Comment (derrix060) [reply resources/i18n/pt_BR/fdmprinter.def.json.po:1446]: I've never seen anyone using `Faz levar` in pt_BR.  I'd still prefer my suggestion for the first part.  I'm okay with the second part though :)

[the remark this answers is not in the document]
Comment (Patola) [reply resources/i18n/pt_BR/fdmprinter.def.json.po:1446]: Beco Sem Saída, a Charlie Brown Jr. song, has this expression (faz levar), and I am from Brasil, so this expression is not pt_PT but pt_BR. Also, 45,000 google hits for the quoted expression.

[the remark this answers is not in the document]
Comment (derrix060) [reply resources/i18n/pt_BR/fdmprinter.def.json.po:1446]: If you want emphasis, you could probably put `Resulta em mais tempo para imptimir, mas torna as superfícies planas mais consistentes.`

**Comment by jellespijker** [comment]
Looks, good. I hope you guys resolved your discussion.  Thank you for contributing.

**Pull request merged**